src/app/pages/dashboard/services/widget-resources.service.ts

```diff
@@ -31,6 +32,9 @@ import { waitForSystemInfo } from 'app/store/system-info/system-info.selectors';
   providedIn: 'root',
 })
 export class WidgetResourcesService {
+  poolStore = globalStore('pool.query');
+  appStore = globalStore('app.query');
```

Comment:
Let's move these to constants somewhere so that new injectors are not created every time.